mne/tests/test_chpi.py

```diff
@@ -222,6 +222,11 @@ def test_calculate_chpi_positions_vv():
     mf_quats = read_head_pos(pos_fname)
     raw = read_raw_fif(chpi_fif_fname, allow_maxshield='yes')
     raw.crop(0, 5).load_data()
+    # check "auto" t_window estimation at full sampling rate
+    with catch_logging() as log:
+        compute_chpi_amplitudes(raw, t_step_min=0.1, t_window='auto',
+                                tmin=0, tmax=2, verbose=True)
+    assert '83.3 ms' in log.getvalue()
```

Comment:
Took me a bit to figure out that this is 5 / 60 because it's the biggest interval (compared to 1 / 23 = 43.4 ms for the delta between 83 and 60; or 5 / 83 = 60.2 ms for the cHPI freqs).

The `5. / f` is motivated by wanting a stable amplitude estimate. The line noise amplitude should be very stable, so I actually wonder if the `5. / f` check should just apply to the cHPI frequencies, and the `1. / delta` to the concat/unique over line+cHPIs...

On the other hand, we do use this for filter_chpi (which does notching at the line freqs), and MaxFilter's default is bigger anyway (0.2), so I can live with it either way. I guess the longer window here is the more conservative choice, so on balance I think I'm +0.5 on the current approach.

Reply:
Sorry, I meant to write some comments but was thrown off by the merge conflicts (I only meant to save to my own fork, forgot the auto stuff).  Perhaps the test I was piggy-backing upon is not robust, as the resulting 83.3 ms time window is trivially determined from the 60 Hz line freq.  Maybe I can temporarily alter the HPI freqs (on a copy of the clipped raw) to apply the proper stresses?

Reply:
No this is good enough, `master` would not use this frequency, so it already demonstrates the change

Reply:
(well, that something was changed, not all the changes -- but good enough for this I think)